kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -323,31 +337,49 @@ define([
         /**
          * Remove the listener for ${msgType} messages for a channel
          *
+         * @param {string} channelType
          * @param {string} channelId
          * @param {string} msgType - the msgType of the listener
          */
-        removeListener(channelId, msgType) {
+        removeListener(channelType, channelId, msgType) {
```

Comment:
edit the removeListener function so that the channelType has to be specified along with the channel ID